blocks/initialization.py

```diff
@@ -223,3 +223,33 @@ def generate(self, rng, shape):
                                                  replace=False)
             weights[i, random_indices] = values[i]
         return weights
+
+
+class Xavier(NdarrayInitialization):
```

Comment:
"Xavier" is the inventor's first name, so it's kind of weird to call it that. "Glorot" or "GlorotBengio" might be a better name...